plugins/MCWhitelist/__init__.py

```diff
@@ -0,0 +1,192 @@
+
+        for entry in whitelist:
+            if entry["name"] == username:
+                return True
+
+        return False
+
+    async def is_disallowed(self, member):
+        query = {"discord_id": member.id}
+        return await self.disallowed_members.find_one(query) != None
+
+    async def remove_whitelisted_account(self, ctx, whitelisted_account):
+        username = whitelisted_account["minecraft_username"]
+        self.whitelist_http_api.remove(username)
+        await self.whitelisted_accounts.delete_many({"discord_id": ctx.author.id})
+
+    async def account_embed(self, whitelisted_account):
+        embed = discord.Embed()
+
+        profile = await self.mojang_api.profile_from_uuid(
+            whitelisted_account["minecraft_uuid"]
+        )
+
+        if profile:
+            skin_url = self.mojang_api.skin_url_from_profile(profile)
+            if skin_url:
+                embed.set_image(url=skin_url)
+
+        username = whitelisted_account["minecraft_username"]
+        embed.colour = discord.Colour.dark_green()
+        embed.add_field(name="Minecraft Username", value=username)
+        return embed
+
+    @commands.group()
+    async def whitelist(self, ctx: commands.Context):
+        """Manage the MUNCS Craft server whitelist."""
+        if not ctx.invoked_subcommand:
+            whitelisted_account = await self.get_whitelisted_account(ctx.author)
+            if whitelisted_account:
+                embed = await self.account_embed(whitelisted_account)
+                await ctx.send(embed=embed)
+            else:
+                await ctx.send(
+                    "You don't have a Minecraft account associated with yourself yet, run !whitelist add <username>."
+                )
+
+    @whitelist.command(name="add")
+    async def whitelist_add(self, ctx: commands.Context, username: str):
+        """Add users to the MUNCS Craft servers whitelist."""
+        if await self.is_disallowed(ctx.author):
+            await ctx.send(
+                f"You are disallowed from being added to the MUNCS Craft whitelist."
+            )
+            return
+
+        whitelisted_account = await self.get_whitelisted_account(ctx.author)
+        if whitelisted_account:
+            username = whitelisted_account["minecraft_username"]
+            await ctx.send(
+                f"You already have the Minecraft account '{username}' associated with your Discord account, run !whitelist remove first to change/add a different one."
+            )
+            return
+
+        verified_role = self.bot.get_guild(PRIMARY_GUILD).get_role(VERIFIED_ROLE)
+        if verified_role not in ctx.author.roles:
+            await ctx.send(
+                "You must first verify your MUN account using !identity before adding yourself to the whitelist."
+            )
+            return
+
+        mojang_resp = self.mojang_api.info_from_username(username)
+        if not mojang_resp:
```

Comment:
Same here, rather than checking if the value is truthy, you should check if the value is None